client/src/components/ContactPage.js

```diff
@@ -1,26 +1,52 @@
-import "../components/About.css"
+import "../components/About.css";
 
 function ContactPage() {
   return (
     <div className="textContainer">
-      <h4>Contact</h4><br></br>
+      <h4>Contact</h4>
+      <br></br>
+      <br></br>
+      <p>HEBESTREIT IMMOBILIEN</p>
+      <br></br>
+      <p>Wildrosenweg 87</p>
+      <br></br>
+      <p>10711 Berlin</p>
       <br></br>
-      <p>HEBESTREIT IMMOBILIEN</p><br></br>
-      <p>Wildrosenweg 87</p><br></br>
-      <p>10711 Berlin</p><br></br>
       <br></br>
       <div className="phoneIconNumber">
-        <svg xmlns="http://www.w3.org/2000/svg" className="svgPhone" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M3 5a2 2 0 012-2h3.28a1 1 0 01.948.684l1.498 4.493a1 1 0 01-.502 1.21l-2.257 1.13a11.042 11.042 0 005.516 5.516l1.13-2.257a1 1 0 011.21-.502l4.493 1.498a1 1 0 01.684.949V19a2 2 0 01-2 2h-1C9.716 21 3 14.284 3 6V5z" />
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          className="svgPhone"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            strokeWidth={2}
+            d="M3 5a2 2 0 012-2h3.28a1 1 0 01.948.684l1.498 4.493a1 1 0 01-.502 1.21l-2.257 1.13a11.042 11.042 0 005.516 5.516l1.13-2.257a1 1 0 011.21-.502l4.493 1.498a1 1 0 01.684.949V19a2 2 0 01-2 2h-1C9.716 21 3 14.284 3 6V5z"
+          />
         </svg>
```

Comment:
These svg files should either be imported as components or an image tag with the src to the svg. This just makes the file unnecessarily long

Reply:
ok I store it in a file